src/components/FilmGallery.vue

```diff
@@ -3,30 +3,68 @@
     <div v-if="!cards || !cards.length" class="no_results_text">
       <span>{{ no_films }}</span>
     </div>
-    <div class="container">
-      <div class="row">
-        <div class="col-md-4" v-for="card in cards" :key="card.title">
-          <film-card class="mb-4" :card="card" />
+    <div v-else>
+      <div class="container">
+        <div class="row">
+          <div class="col-md-4" v-for="card in cards" :key="card.title">
+            <film-card class="mb-4" :card="card" />
+          </div>
         </div>
       </div>
+      <div class="load_more text-center">
+        <load-more-button
+          v-if="!pageIsFullyLoaded"
+          @loadMore="loadMore"
```

Comment:
Consult https://vuejs.org/v2/api/#v-on
"Shorthand Dynamic Event"
Your goal is to change @loadMore to something that corresponds to the event variable you're emitting.